getting-started/bookshelf/db_memory.go

```diff
@@ -12,58 +12,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package bookshelf
+package main
 
 import (
+	"context"
 	"errors"
 	"fmt"
 	"sort"
+	"strconv"
 	"sync"
 )
 
-// Ensure memoryDB conforms to the BookDatabase interface.
 var _ BookDatabase = &memoryDB{}
 
 // memoryDB is a simple in-memory persistence layer for books.
 type memoryDB struct {
 	mu     sync.Mutex
-	nextID int64           // next ID to assign to a book.
-	books  map[int64]*Book // maps from Book ID to Book.
+	nextID int64            // next ID to assign to a book.
+	books  map[string]*Book // maps from Book ID to Book.
 }
 
 func newMemoryDB() *memoryDB {
```

Comment:
memoryDB is not used anymore either

Reply:
but maybe it should be (for tests? see comment below)

Reply:
Added to tests - see other comment.